static/styles/left_sidebar.css

```diff
@@ -190,25 +191,96 @@ li.show-more-topics {
     }
 }
 
-#private-container,
-#stream-filters-container {
+#left_sidebar_scroll_container {
+    outline: none;
     overflow-x: hidden;
     overflow-y: auto;
     position: relative;
     z-index: 0;
     width: 100%;
 }
 
-#stream-filters-container .simplebar-content-wrapper {
-    outline: none;
+#private_messages_container {
+    margin-bottom: $sections_vertical_gutter;
+    margin-right: 12px;
+
+    #toggle_private_messages_section_icon {
+        opacity: 0.7;
+        margin-left: 3px;
+        min-width: 13px;
+    }
+
+    #private_messages_section_header {
+        cursor: pointer;
+        padding-left: $far_left_gutter_size;
+        padding-right: 10px;
+        white-space: nowrap;
+
+        #show_all_private_messages {
+            right: 0;
+            float: right;
+            position: absolute;
+            opacity: 0.7;
+            text-decoration: none;
+            color: inherit;
+            margin-right: 25px;
```

Comment:
Are the `10px` right padding, the `25px` here and the `20px` right margin for the unread count just below related to any of the other numbers in this file?